README.md

```diff
@@ -0,0 +1,62 @@
+# Fedora CI pipeline library
+
+This git repository contains a library of reusable [Jenkins Pipeline](https://jenkins.io/doc/book/pipeline/) steps and functions that can be used in your Jenkinsfile to make your Fedora CI pipelines simple.
+
+## How to use this library
+
+This library is intended to be used as a [global shared library](https://jenkins.io/doc/book/pipeline/shared-libraries/#global-shared-libraries) configured in your Jenkins instance.
+
+It is recommended to pin down a specific version of the library and update it cautiously.
+
+To use the steps in this library just add the following to the top of your Jenkinsfile:
+
+```jenkinsfile
+@Library('fedora-pipeline-library') _
+```
+
+The snippet above assumes that you decided to use the name `fedora-pipeline-library` when configuring the library in the global settings.
+
+## Steps
+
+This is a list of all steps implemented in the library. Click on a particular step to find out more about it.
+
+* [abort()](./docs/steps/abort.md)
+* [buildImageAndPushToRegistry()](./docs/steps/buildImageAndPushToRegistry.md)
+* [getGitUrl()](./docs/steps/getGitUrl.md)
+* [getReleaseIdFromBranch()](./docs/steps/getReleaseIdFromBranch.md)
+* [isProduction()](./docs/steps/isProduction.md)
+* [isPullRequest()](./docs/steps/isPullRequest.md)
+* [sendMessage()](./docs/steps/sendMessage.md)
+* [setBuildNameFromArtifactId()](./docs/steps/setBuildNameFromArtifactId.md)
+
+## Limitations
+
+Fedora CI is "artifact-centric", i.e. everything more or less evolves around artifacts. This library currently supports only "koji-build" artifacts, modules and other artifact types are not supported yet.
```

Comment:
I think this requires a semi-colon:

"This library currently supports only "koji-build" artifacts; modules and other artifact types are not supported yet."

Reply:
Updated. Thanks! :wink: